service/src/main/java/com/sedliarov/learningtable/service/StudentService.java

```diff
@@ -18,6 +18,8 @@ public interface StudentService {
 
   void deleteStudent(UUID id);
 
+  void deleteStudents();
```

Comment:
We don't use this in controller and my question, For what we have this method?

Reply:
Done